### PR TITLE
Adding --list_resources to Inventory

### PR DESCRIPTION
**Pull request by mirons-google**
This consolidates the configuration file to the samples directory. It also organizes the resources in the configuration file. This separation will be useful once all the inventory flags are moved to the inventory_conf.yaml.sample

### CODE REVIEW

File: google/cloud/security/inventory/inventory_loader.py [hunk 1]
```diff
@@ -274,14 +275,19 @@ def _create_dao_map():
...
     inventory_flags = FLAGS.FlagValuesDict()
+
+    if inventory_flags.get('list_resources'):
+        inventory_util.list_resource_pipelines()
+        sys.exit()
+
     _configure_logging(inventory_flags.get('loglevel'))
 
-    if inventory_flags.get('config_path') is None:
+    inventory_conf = inventory_flags.get('config_path')
```
Comment (blueandgold) [inline google/cloud/security/inventory/inventory_loader.py]: nit: Can we make ```inventory_conf``` to be ```config_path``` to be consistent?  So, this line would be:
```python
config_path = inventory_flags.get('config_path')
```

File: samples/inventory/inventory_conf.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Google Inc.
```
Comment (blueandgold) [inline samples/inventory/inventory_conf.yaml]: QQ: If this is in samples/ now, where should it be when actually deployed?

File: google/cloud/security/inventory/inventory_loader.py (hunk 1 above)
Comment (mirons-google) [inline google/cloud/security/inventory/inventory_loader.py]: done!

File: samples/inventory/inventory_conf.yaml (hunk 2 above)
Comment (mirons-google) [inline samples/inventory/inventory_conf.yaml]: Discussed offline with @blueandgold on possible deployment options.

**Comment by blueandgold** [comment]
LGTM on the changes

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**